controllers.topic/topic.controller.js

```diff
@@ -0,0 +1,11 @@
+const readTopics  = require('../models.topic/topic.model');
+
+const getTopics = (request, response) => {
+    readTopics()
+    .then((topics) => {
+        response.status(200).send(topics)
+    })
```

Comment:
This is really good, just one thing to add is we want to always send an object in the response `.send` , to make this an object with a key of _topics_ you can write `.send({ topics })` 

This will mean some changes need to be made in the test also to access the data